### PR TITLE
ISS-22784: React Native iOS PIP View

### PR DESCRIPTION
**Pull request by ygit**
# Description

- ISS-22784: React Native iOS PIP View

### Pre-launch Checklist

- [x] The [Documentation] is updated accordingly, or this PR doesn't require it.
- [x] I have updated the `ExampleAppChangelog.txt` file with relevant changes.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making, or this PR is test-exempt.
- [x] All existing and new tests are passing.

[Documentation]: https://www.100ms.live/docs

### CODE REVIEW

File: packages/react-native-hms/ios/HMSRNSDK.swift [hunk 1]
```diff
@@ -1457,6 +1460,15 @@ class HMSRNSDK: HMSUpdateListener, HMSPreviewListener {
         }
         let roomData = HMSDecoder.getHmsRoomSubset(room)
         self.delegate?.emitEvent(HMSConstants.ON_JOIN, ["event": HMSConstants.ON_JOIN, "id": self.id, "room": roomData])
+
```
Comment (stanwolverine) [inline packages/react-native-hms/ios/HMSRNSDK.swift:1463]: I think we can delete this code block

**Review by stanwolverine — CHANGES_REQUESTED** [review]
I noticed that HMSIOSPIPController code is still there, please confirm and remove it if we are not using

File: packages/react-native-hms/ios/HMSRNSDK.swift (hunk 1 above)
Comment (ygit) [reply packages/react-native-hms/ios/HMSRNSDK.swift:1463]: Done

**Review by stanwolverine — APPROVED** [review]
LGTM

**Pull request merged**